.circleci/config.yml

```diff
@@ -274,15 +274,15 @@ jobs:
             if [ "${CIRCLE_PROJECT_USERNAME}" == "replicatedhq" ]; then
               sed -i "s/DYNAMIC_IMAGE_TAG/${CIRCLE_SHA1:0:7}/g" ~/project/kustomize/overlays/production/kustomization.yaml
               echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==' >> ~/.ssh/known_hosts
-              cd ~ && git clone git@github.com:replicatedcom/gitops-deploy
+              cd ~ && git clone --single-branch -b release git@github.com:replicatedcom/gitops-deploy
```

Comment:
is there a reason not to `--single-branch -b master` on the staging deploy?

Reply:
@dexhorthy i copied this from vendor api
https://github.com/replicatedcom/vendor-api/blob/master/.circleci/config.yml#L122